content/en/_index.html

```diff
@@ -3,26 +3,138 @@
+	<br/><br/>
+</div>
+<div>
+	CDEvents is a common specification for Continuous Delivery events, enabling interoperability in the complete software production ecosystem.
+	<br/>
+</div>
 <div class="mx-auto mt-5">
 	{{< blocks/link-down color="info" >}}
 </div>
 {{< /blocks/cover >}}
 
-{{< blocks/lead title="About CDEvents" color="dark">}}
+{{< blocks/lead title="About CDEvents" color="white">}}
+<p>
+	We are all too aware that teams building software products have a hidden overhead that they must manage. Everyone involved understands that you are building a machine to solve your customers' problem, and that this machine is an asset that must be invested in and maintained, in order to maximize its value. What many fewer people recognize is that to do this successfully, you must also own a second machine, whose only purpose is to build the first.
+</p>
+<div>
+	<p>The first machine, that is your product, is highly visible and commands all the attention and investment.</p>
+	<p>The second machine, that is critical to maintaining the value of the first, often sits in darkness, neglected and poorly understood, despite being essential to building all the value in your business.</p>
+</div>
+<p>
+	When you design your product, you have the freedom to choose from an almost infinite palette of reusable, modular libraries and frameworks. When you design your build system, however, your choices are much more limited, often being constrained to accepting the features offered by your current cloud vendor, or building something from first principles, which must be maintained with scant resources.
+</p>
+<p>
+	Although many tools exist in the CI/CD space, they have been unable to talk to each other in any standardized way, until today. The Continuous Delivery Foundation’s CDEvents project has been created to solve this interoperability problem, providing a new evolution in CI/CD capabilities.
+</p>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="CDF" color="light">}}
+CDEvents is an incubated project at the <a href="https://cd.foundation/projects">Continuous Delivery Foundation</a>
+
+<a href="https://cd.foundation/"><img src="https://raw.githubusercontent.com/cdfoundation/artwork/main/cdf/stacked/color/cdf-stacked-color.svg" width="20%"/></a>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="Benefits" color="white">}}
+<p>
+	CDEvents builds upon the work done by the CDF to set out <a href="https://bestpractices.cd.foundation">best practices in Continuous Delivery</a>, to define a common language for CI/CD ecosystem events, permitting the decoupling of pipeline descriptions from physical implementations. A decoupled CI/CD architecture is easy to scale and makes the CI/CD pipelines more resilient
+	to failures, which is critical as the end-to-end software production and delivery pipelines grow more and more complex, not least in a microservices architecture with thousands of independent pipelines. 
```

Comment:
I miss one aspect of CDEvents here. In this section, and also in previous sections, the focus is only on decoupling sub-pipelines in an end-to-end pipeline from each other, so that they can be managed by different tooling.

I'd also like to see here, and above, statements about events sent by CI/CD orchestrators and consumed by other kinds of tools like metrics calculators and pipeline visualizers.

The interoperability could maybe be seen in two dimensions - one horizontal dimension where different CI/CD orchestrators interoperate, and one vertical where consumers of the events from those orchestrators make additional use of the data represented by the events.

This is mentioned further down I see now, describing the primary and secondary use cases there, but I think both those use cases should be included in the top most sections of this page as well somehow. We'd like to attract not only CI/CD orchestration tool developers, but also people in the observability, metrics, visualization and AI/ML spaces etc

Reply:
I'm not sure I understand what you are describing, which suggests to me that this should be treated in an expanded narrative in the Primer, where you can clearly illustrate with examples.

Reply:
The second use case in https://github.com/cdevents/spec/blob/main/primer.md#use-cases

Reply:
Observability is mentioned in the last paragraph of this section now, so I believe we could consider that use case covered.